src/main/java/com/palominolabs/metrics/guice/intercept/CountedInterceptor.java

```diff
@@ -1,16 +1,16 @@
-package com.palominolabs.metrics.guice;
+package com.palominolabs.metrics.guice.intercept;
 
 import com.codahale.metrics.Counter;
 import com.codahale.metrics.annotation.Counted;
 import org.aopalliance.intercept.MethodInterceptor;
 import org.aopalliance.intercept.MethodInvocation;
 
-class CountedInterceptor implements MethodInterceptor {
+public class CountedInterceptor implements MethodInterceptor {
```

Comment:
As I see it, these are pretty generic interceptors for metrics framework, that can be reused by the clients of the library. Why do you want to keep them private?